### DISCUSSION TITLE
tok2vec vs word2vec

### DISCUSSION THREAD

**Discussion by iyedbennour** [explosion/spaCy#8690] [Help: Other Questions] [answered]
Sorry if my question is too obvious but I can't find any information about tok2vec and how it's related to word2vec. Is there is any description of this model somewhere ? 
Thank you !

**Comment by polm** [comment, marked as answer]
See [the architecture overview](https://spacy.io/api/architectures). 

tok2vec is not directly related to word2vec - xxx2vec is a common naming scheme for models or systems since word2vec, even if they don't have much in common architecturally. tok2vec isn't exactly a single model either, it's just the name of the layer in spaCy that converts tokens into vectors. The default models are described on the architecture overview page.
